docs/install.md

```diff
@@ -131,27 +131,44 @@ page. Copy the contents of the SPID into the file
 
 SGX can run in either simulation or hardware mode. No kernel driver is
 required to run in simulation mode. However, if you plan to run with SGX
-hardware support, it is necessary to install the SGX kernel driver. The
+hardware support, it is necessary to install the SGX kernel driver. 
+
+<!-- DCAP kernel driver installation -->
+The following commands will download and install the driver version 1.3 of
+the DCAP SGX kernel driver (for Ubuntu 18.04 server):
+
+```bash
+DRIVER_REPO=https://download.01.org/intel-sgx/sgx-dcap/1.7/linux/distro/ubuntu18.04-server/
+DRIVER_FILE=sgx_linux_x64_driver_1.35.bin
+
+wget ${DRIVER_REPO}/${DRIVER_FILE} -P /tmp
+chmod a+x /tmp/${DRIVER_FILE}
+sudo ./${DRIVER_FILE}
+```
+Note:
+- as of early August 2020, the sdk drivers (e.g., `sgx_linux_x64_driver_2.6.0_602374c.bin`) will cause BUS errors in some circumstance and is currently not supported by PDO
```

Comment:
Even the previous SDK driver 26_95eaa6f has the same issues, not sure if we should document the earliest one that we had problems with.

Reply:
yes, indeed moving to earlier sdk drivers doesn't work (at least the three preceding ones didn't help), that's why i added the specific one only as `e.g.'.   Not sure if any of the even earlier ones still would work with the new debian packages?  If you know one, then of course would be great if we could document that.  But not sure if it is worth spending a lot of time on it to figure out a config?

Reply:
Right yeah, I don't think it's worth our time trying to find a working config with an SDK driver. I haven't tested any earlier SDK drivers with the current debian packages. Maybe a good way to document this is to specify the driver versions for which we have confirmed that the SDK 2.9+ debian packages cause this issue on FLC hardware?

Reply:
Given that it _does_ work with sdk driver on non-FLC HW, i changed section to describe both driver installations and their properties